.github/workflows/release-artifacts.yml

```diff
@@ -19,6 +19,9 @@ jobs:
 
           - target: aarch64-unknown-linux-musl
             os: ubuntu-latest
+
+          - target: aarch64-unknown-linux-musl
```

Comment:
What's the difference between line 20, where `aarch64-unknown-linux-musl` is already defined?

Reply:
Yeah, I'm kind of surprised by this, but it appears to have been implemented already: https://github.com/qdrant/qdrant/issues/3714#issuecomment-2009330629